modules/identity-provider-oidc-supported.adoc

```diff
@@ -5,7 +5,7 @@
 [id="identity-provider-oidc-supported_{context}"]
 = Supported OIDC providers
 
-The following OpenID Connect (OIDC) providers are tested and supported with {product-title}:
+Red Hat tests and supports certain OpenID Connect (OIDC) providers with {product-title}. The following OpenID Connect (OIDC) providers are tested and supported with {product-title}. Using an OIDC provider that is not on the following list might work with {product-title}, but the provider was not tested by Red Hat and therefore is not supported by Red Hat.
```

Comment:
I know with support language it is kind of tricky changing things. I wonder though if changing `certain` to `specific` gets us more clarity. Certain seems a bit open for interpretation where you've got a very specific list.